nni/nas/execution/common/integration.py

```diff
@@ -239,4 +256,6 @@ def handle_import_data(self, data):
         pass
 
     def handle_add_customized_trial(self, data):
-        pass
+        previous_max_param_id = self.recover_parameter_id(data)
+        self.parameters_count = previous_max_param_id
+        self._advisor_initialized = True
```

Comment:
Is it possible that `handld_add_customized_trial` is never called and `_advisor_initialized` is never set to true?

Reply:
good point! `handle_add_customized_trial` is called when experiment is resumed, but will not be called when experiment is created, this is a bug introduced by me...
I moved this flag to `handle_request_trial_jobs`, which means if trial is not requested, `send_trial` will be blocked. And "request trial" will always be sent by nnimanager